packages/shelter-ui/package.json

```diff
@@ -14,6 +14,8 @@
   "devDependencies": {
     "esbuild": "^0.20.1",
     "esbuild-sass-plugin-ysink": "^2.3.3",
+    "postcss": "^8.4.35",
+    "postcss-modules": "^6.0.0",
```

Comment:
is this necessary?? I haven't found it to be so but that might be luck. it is out of scope of this PR however.

Reply:
also, I can't build without these changes.

Reply:
that's fair, I'll fix this, but it shouldn't be here